Formula/gnupg.rb

```diff
@@ -42,7 +42,8 @@ def install
   end
 
   def caveats; <<-EOS.undent
-    All commands have been installed with the suffix '1'.
+    This formula does not install either `gpg` or `gpgv` vanilla executables
+    into the PATH. All executables have been installed with a "1" suffix.
```

Comment:
👍 on vanilla, will reword this blob a bit.

> Exciting changes afoot!

Hopefully. At this point we're diving into parts that weren't working entirely as expected when I was trying to blob everything together in one big PR, so I'm kind of frantically running around trying to find the root cause of why that was happening now there's less changes ongoing around it 🙈.